data/en/ecommerce_0001.json

```diff
@@ -2132,8 +2132,8 @@
                                 "label": "",
                                 "description": "",
                                 "options": [{
-                                        "label": "They’re not used",
```

Comment:
Hmm, this will affect SDX. Is it only this one?

Reply:
This is how it was originally in the slides...

Reply:
Ah